shop/src/main/java/ro/msg/learning/shop/Repository/ILocationRepository.java

```diff
@@ -0,0 +1,15 @@
+package ro.msg.learning.shop.Repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import ro.msg.learning.shop.Domain.Customer;
+import ro.msg.learning.shop.Domain.Location;
+
+import java.util.UUID;
+
+@Repository
+public interface ILocationRepository extends JpaRepository<Location, Long> {
```

Comment:
I think we can use `public interface ILocationRepository extends JpaRepository<Location, UUID> {`. Please check.